src/pack/rpm.go

```diff
@@ -0,0 +1,68 @@
+package pack
+
+import (
+	"fmt"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/src/project"
+	"github.com/tarantool/cartridge-cli/src/rpm"
+)
+
+func packRpm(projectCtx *project.ProjectCtx) error {
+	// check context
+	if err := checkPackRpmCtx(projectCtx); err != nil {
+		// TODO: format internal error
+		panic(err)
+	}
+
+	// app dir
+	appDirPath := filepath.Join(projectCtx.PackageFilesDir, "/usr/share/tarantool/", projectCtx.Name)
+	if err := initAppDir(appDirPath, projectCtx); err != nil {
+		return err
+	}
+
+	// systemd dir
+	if err := initSystemdDir(projectCtx.PackageFilesDir, projectCtx); err != nil {
+		return err
+	}
+
+	// tmpfiles dir
+	if err := initTmpfilesDir(projectCtx.PackageFilesDir, projectCtx); err != nil {
+		return err
+	}
+
+	// construct RPM file
+	if err := rpm.Pack(projectCtx); err != nil {
+		return err
+	}
+
+	log.Infof("Created result package: %s", projectCtx.ResPackagePath)
+
+	return nil
+}
+
+func checkPackRpmCtx(projectCtx *project.ProjectCtx) error {
+	if projectCtx.Version == "" {
+		return fmt.Errorf("Missed project version")
```

Comment:
It doesn't look like a statement. I think "Project version is missed". But may be I'm wrong - ask somebody more competent.

Reply:
It's relevant for all messages below.

Reply:
Yes, I changed it to `Version is missed` etc.